notification2-examples/shared-components/src/main/java/c8y/example/notification/common/platform/SubscriptionRepository.java

```diff
@@ -1,16 +1,12 @@
-package c8y.example.notification.helloworld.platform;
+package c8y.example.notification.common.platform;
 
-import com.cumulocity.model.idtype.GId;
 import com.cumulocity.rest.representation.reliable.notification.NotificationSubscriptionRepresentation;
 import com.cumulocity.sdk.client.messaging.notifications.NotificationSubscriptionApi;
 import com.cumulocity.sdk.client.messaging.notifications.NotificationSubscriptionCollection;
 import com.cumulocity.sdk.client.messaging.notifications.NotificationSubscriptionFilter;
 import lombok.RequiredArgsConstructor;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Component;
 
-@Component
-@RequiredArgsConstructor(onConstructor_ = @Autowired)
+@RequiredArgsConstructor
 public class SubscriptionRepository {
```

Comment:
Not really in scope of this change but what do you think about renaming to `SubscriptionRepository` so the naming is consistent with `TokenService`?

Reply:
Oh, I must have named them so to reflect their state - subscriptions are persisted where as tokens are generated and are not stored anywhere.